bluebottle/funding/effects.py

```diff
@@ -22,7 +22,10 @@ class GeneratePayoutsEffect(Effect):
 
     def post_save(self, **kwargs):
         from bluebottle.funding.models import Payout
-        Payout.generate(self.instance)
+        try:
+            Payout.generate(self.instance)
+        except AssertionError:
+            pass
```

Comment:
Why this?

Reply:
There was at least one old activity that changed status because of the save. This triggered this effect, however there already was a payout generated for this activity. This makes sures that the effect still works even if the check fails.